content/en/account_management/scim/entra.md

```diff
@@ -48,20 +51,20 @@ When using SAML and SCIM together, Datadog strongly recommends disabling SAML ju
 ### User attributes
 
 1. Expand the **Mappings** section
-2. Click **Provision Azure Active Directory Users**
+2. Click **Provision Azure Active Directory Users**. The Attribute Mapping page appears.
```

Comment:
Note that this screen still says Azure Active Directory. Microsoft has not yet updated the app to say Entra ID.